css/styles.css

```diff
@@ -321,3 +472,451 @@ textarea {
   left: 4px;
   font-size: 11px;
 }
+
+.desktopnav {
+  visibility: hidden;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+  border-bottom-width: 1px;
+  border-bottom-style: solid;
+  border-bottom-color: lightgray;
+  display: flex;
+  justify-content: center;
+}
+
+.desktopnav ul li {
+  margin-right: 30px;
+}
+
+.awesomedeco1 {
+  background-image: url(../resources/image-geometry_7.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  left: 1488px;
+  top: 296px;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+}
+
+.awesomedeco2 {
+  background-image: url(../resources/image-geometry_6.svg);
+  background-repeat: no-repeat;
+  position: absolute;
+  visibility: hidden;
+  width: 353px;
+  height: 351px;
+  left: -86px;
+}
+
+.imggeo2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imggeo5 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.fixGap1 {
+  margin-top: -1px;
+  margin-bottom: -5px;
+}
+
+.imgGroup83 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.redSquare {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.imageDots2 {
+  height: 0;
+  width: 0;
+  visibility: hidden;
+}
+
+.dflex3 {
+  display: flex;
+  justify-content: center;
+}
+
+.pad-right {
+  padding-right: 21px;
+}
+
+@media screen and (min-width: 1680px) {
```

Comment:
- [ ] I really appreciate your effort in implementing the changes suggested by the previous reviewers. I really excuse for the inconvenience **but truly the `min-width` you set for the desktop screen is too big that no one with a standard PC can be able to test it.** Among the project guideline, **You are expected to add the min-width that is in the range of 768px - 1440px.**  Please I humbly request that you add the min-width in that range so that the next reviewer will be able to check what your desktop version looks like 😃 😎 👍

Reply:
I have no problem complying. And I totally understand your point. The reason I used 1680px is because that is the size for the desktop given to us by this article https://medium.com/level-up-web/best-practices-of-responsive-web-design-6da8578f65c4 (provided by Microverse). It would have been ideal to know from the beginning that I should have used another size. I will use 1440px, I hope there is no prob with that. Thank you so much for your help, and have a great day!